tests/OmniSharp.Script.Tests/WorkspaceInformationTests.cs

```diff
@@ -77,35 +77,44 @@ public async Task SingleCsiScriptWithCustomRspNamespacesAndReferences()
         public async Task CsiScriptWithFileCreatedAfterStartingServer()
         {
             using (var testProject = TestAssets.Instance.GetTestScript("EmptyScript"))
-            using (var host = CreateOmniSharpHost(testProject.Directory))
             {
-                var workspaceInfo = await GetWorkspaceInfoAsync(host);
-                Assert.Empty(workspaceInfo.Projects);
+                using (var host = CreateOmniSharpHost(testProject.Directory))
```

Comment:
why is this test changed?

Reply:
it actually changes the intent of the test, the intent was:
1) initialize workspace to some directory
2) verify no script files are there
3) create a file *after* the host is running (now with the change, the file is created *before* the host is running)
4) verify that the script file was picked up by a running host

Reply:
In my environment, the original CsiScriptWithFileCreatedAfterStartingServer did not pass.
![capture1](https://user-images.githubusercontent.com/5210329/61799896-ec8be880-ae66-11e9-88c1-b0afa87ee923.jpg)
![capture2](https://user-images.githubusercontent.com/5210329/61799907-f01f6f80-ae66-11e9-9677-723f4e42bfff.jpg)

Reply:
thanks for finding this. see https://github.com/OmniSharp/omnisharp-roslyn/pull/1570
once that is merged, you can rebase on top of latest master and everything should be fine without any need to change this test anymore.

Reply:
Thank you @filipw.